setup_wordpress.sh

```diff
@@ -4,3 +4,68 @@ set -e
 
 # Your code goes below ...
 echo 'This script should install and setup Wordpress'
+
+echo '---------- Update & Upgrade APT ----------'
+sudo apt-get update -y
+sudo apt-get upgrade -y
+
+echo '---------- Install Nginx Package and configure Firewall ----------'
+sudo apt-get install nginx -y
+sudo ufw allow 'Nginx Full'
+
+echo '---------- Install MYSQL Packages ----------'
+sudo apt-get install mysql-server -y
+
+echo '---------- Create WordPress Database & User ----------'
+mysql_pw='YourPassword123!'
+sudo debconf-set-selections <<< "mysql-server mysql-server/root_password password ${mysql_pw}"
+sudo debconf-set-selections <<< "mysql-server mysql-server/root_password_again password ${mysql_pw}"
+sudo mysql -u root -p$mysql_pw < /vagrant/wpDB.sql
+
+echo '---------- Install PHP Packages ----------'
+sudo apt install php7.4-cli php7.4-fpm php7.4-mysql php7.4-json php7.4-opcache php7.4-mbstring php7.4-xml php7.4-gd php7.4-curl -y
+
+echo '---------- Self-SSL Certificate ----------'
+sudo openssl req -x509 -newkey rsa:2048 -days 365 -nodes \
```

Comment:
could you give more details about this self signed cert?

Reply:
-openssl = cm for creating and managing OpenSSL
-req: subcm that specifies use of X.509 Certificate singing request. "X.509" public key infrastructure.
-new key rsa:2048: specifies that we want to generate a new certificate and a new key at the same time, 2048bits long
-days 365: lenght of time cert will be considered valid
-nodes = tells openssl to skip the option to secure our certificate with a passphrase. Lets ngninx read the file without user intervention.